src/stateMachines/deactivation.ts

```diff
@@ -174,7 +174,7 @@ export class Deactivation {
   async invokeUnprovision (context: DeactivationContext): Promise<any> {
     let { message, clientId } = context
     const clientObj = devices[clientId]
-    const xmlRequestBody = this.amt.SetupAndConfigurationService(AMT.Methods.UNPROVISION, null, 2)
+    const xmlRequestBody = this.amt.SetupAndConfigurationService.Unprovision(1)
```

Comment:
Do we always pass value 1 for both CCM and ACM deactivation/unprovision?

Reply:
per the AMT SDK: ProvisioningMode Indicates the provisioning mode (Enterprise , Small Business or Remote Connectivity) the device will enter following successful completion of the command. Starting from Release 6.0 only effective value is ProvisioningModeEnterprise

ProvisioningModeCurrent: 0
ProvisioningModeEnterprise: 1
ProvisioningModeSmallBusiness: 2
ProvisioningRemoteConnectivity: 3